From 26c52ffc8b389c5866bef097398cf2e1e9198fcb Mon Sep 17 00:00:00 2001
From: Irina Tirdea <irina.tirdea@nxp.com>
Date: Tue, 14 Feb 2017 10:37:56 +0200
Subject: [PATCH 4897/5242] MLK-13869 Revert "ARM: imx: add sleep for pllv3
 relock"

commit  031191234e256ad63274f391199bb54dd68ed9f7 from
https://source.codeaurora.org/external/imx/linux-imx.git

This reverts commit 322503a15740bd9383bb4ed452e5dd5a40598170.
The driver for clk-pllv3 has moved from arch/arm/mach-imx/clk-pllv3.c
to drivers/clk/imx/clk-pllv3.c since the orginal change was made,
so the revert is done to the new file instead.

Signed-off-by: Irina Tirdea <irina.tirdea@nxp.com>
(cherry picked from commit dd50ef8f53be467f59947e4f2b3d03c093ec9783)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-pllv3.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/clk/imx/clk-pllv3.c b/drivers/clk/imx/clk-pllv3.c
index 1ae8226..2500353 100644
--- a/drivers/clk/imx/clk-pllv3.c
+++ b/drivers/clk/imx/clk-pllv3.c
@@ -11,7 +11,6 @@
  */
 
 #include <linux/clk-provider.h>
-#include <linux/delay.h>
 #include <linux/imx_sema4.h>
 #include <linux/io.h>
 #include <linux/slab.h>
@@ -76,7 +75,6 @@ static int clk_pllv3_wait_lock(struct clk_pllv3 *pll)
 			break;
 		if (time_after(jiffies, timeout))
 			break;
-		usleep_range(50, 500);
 	} while (1);
 
 	return readl_relaxed(pll->base) & BM_PLL_LOCK ? 0 : -ETIMEDOUT;
-- 
1.7.9.5

